spellbook/dbt_project.yml

```diff
@@ -127,4 +127,10 @@ on-run-end:
   - "{{ alter_tblproperties_opensea_trades() }}"
   - "{{ alter_tblproperties_opensea_volume_day() }}"
   - "{{ alter_tblproperties_opensea_txns_day() }}"
-  - "{{ alter_tblproperties_opensea_active_traders_day() }}"
\ No newline at end of file
+  - "{{ alter_tblproperties_opensea_active_traders_day() }}"
+
+  - "OPTIMIZE transfers_ethereum.erc20_agg_hour"
```

Comment:
These commands are gonna run on create right? But will they also run on incremental runs? Ideally these should periodically. For context, optimize for prices run hourly (but inserts happen every 5 min or so there).

Reply:
These commands are actually ran every time a incremental run updates as well (so quarter hourly for now)